### PR TITLE
SPOI-4706: Fix names and description of properties in R operator

### PR DESCRIPTION
**Pull request by sandeepdeshmukh**
Please review.

### CODE REVIEW

**Comment by pm48** [comment]
Please do a squash commit.

**Comment by pm48** [comment]
Created jira:MLHR-1740 for above, please refer the jira here.

**Comment by sandeepdeshmukh** [comment]
Squashed.

File: contrib/src/main/java/com/datatorrent/contrib/r/RScript.java [hunk 1]
```diff
@@ -38,35 +38,42 @@
...
  * form of a function. This function will then be called by the operator.
  *
- * The user should - 1. set the name of the script file (which contains the script in the form of a function) 2. set the
- * function name. 3. set the name of the return variable 4. Make sure that the script file is available in the
- * classpath. 5. set the type of arguments being passed. This will be done in a Map. 6. Send the data in the form of a
- * tuple consisting of a key:value pair where, "key" represents the name of the argument "value" represents the actual
- * value of the argument. A map of all the arguments is created and passed as input. The result will be returned on one
- * of the output ports depending on the type of the return value.
+ * The user should <br>
+ * 1. set the name of the script file (which contains the script in the form of a function) <br>
```
Comment (243826) [inline contrib/src/main/java/com/datatorrent/contrib/r/RScript.java]: <ol> tag in HTML?

**Comment by 243826** [comment]
Looks like this PR has conflicts. Can you send a fresh one in against the latest?

**Pull request closed without merging**